docs/source/examples/neat_projects.rst

```diff
@@ -0,0 +1,13 @@
+NeatProjects
+=================================
+
+.. manim:: Projects11
+    :quality: medium
+    :save_last_frame:
+
+    class Projects11(Scene):
+        def construct(self):
+            t = TextMobject("Some neat project")
+            self.add(t)
+            self.wait(1)
```

Comment:
Same comment as for `advanced_projects.rst`.

```suggestion
This page is currently under construction. It will feature a selection of neat projects built with manim.
```